hooks/useMemesQuickVoteSessionState.ts

```
@@ -0,0 +1,496 @@
+      } catch {
+        if (
+          abortController.signal.aborted ||
+          syncRequestIdRef.current !== requestId
+        ) {
+          return;
+        }
+
+        setCurrentSessionState(getQuickVoteSyncFailureState);
+      } finally {
+        if (syncRequestIdRef.current !== requestId) {
+          return;
```

Biome (2.5.3)

[error] Unsafe usage of 'return'.

(lint/correctness/noUnsafeFinally)